components/playlist/browser/playlist_download_request_manager.cc

```diff
@@ -292,7 +292,7 @@ void PlaylistDownloadRequestManager::ProcessFoundMedia(
     // nullable data
     auto* thumbnail = media_dict.FindString("thumbnail");
     auto* author = media_dict.FindString("author");
-    auto duration = media_dict.FindInt("duration");
+    auto duration = media_dict.FindDouble("duration");
```

Comment:
I realized that this was changed from double to int when I got feedback about conversion. https://github.com/brave/brave-core/pull/16419#discussion_r1061058410